src/olympia/accounts/views.py

```diff
@@ -144,12 +144,9 @@ def find_user(identity):
         return user
     except UserProfile.DoesNotExist:
         return None
-    except UserProfile.MultipleObjectsReturned:
-        # This shouldn't happen, so let it raise.
-        log.error(
-            'Found multiple users for %s and %s', identity['email'], identity['uid']
-        )
-        raise
+    # UserProfile.MultipleObjectsReturned might be raised here. This is
+    # expected, we don't know what to do in this case so we let the exception
+    # happen loudly.
```

Comment:
Might it still be better to raise a custom error/exception with useful information in the string?

Reply:
+1. We are still getting inquiries about this from affected developers occasionally. Could we include the two user ids here?

Reply:
The traceback already contains this:
```clone: <BaseQuerySet [<UserProfile: 1234567: Foo>, <UserProfile: 1234568: Bar>]>```

Reply:
Ah, thanks, that wfm.